batch7crud-roll9/src/main/java/com/lftechnology/batch7crud/util/StudentValidator.java

```diff
@@ -0,0 +1,77 @@
+package com.lftechnology.batch7crud.util;
+
+import com.lftechnology.batch7crud.constants.AppConstant;
+import com.lftechnology.batch7crud.constants.AttributeConstant;
+import com.lftechnology.batch7crud.constants.MessageConstant;
+import com.lftechnology.batch7crud.constants.ParameterConstant;
+import com.lftechnology.batch7crud.entity.Student;
+
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Created by sanjay on 1/22/16.
+ */
+public class StudentValidator {
+  public Student createObject(HashMap error, HashMap input) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The type of the "input" object should be an interface such as "Map" rather than the implementation "HashMap". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1319)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The type of the "error" object should be an interface such as "Map" rather than the implementation "HashMap". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1319)